sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -308,7 +295,10 @@ export enum PiiEntityDomainType {
 }
 
 // @public
-export type PiiTaskParametersDomain = "phi" | "none" | string;
+export type RecognizeCategorizedEntitiesAction = {
```

Comment:
Java had GAed `RecognizeEntitiesResult` for the NER (which is the categorized entities). So by follow up the same name pattern, java is using `RecognizeEntitiesAction`

Reply:
Ok but JS uses the Categorized prefix in its GA.